README.md

```diff
@@ -1,9 +1,9 @@
-# Check Protocols
-[![CircleCI](https://circleci.com/gh/Originate/check-protocols.svg?style=svg)](https://circleci.com/gh/Originate/check-protocols)
-[![open issues](https://img.shields.io/github/issues/originate/check-protocols.svg?label=open%20issues)](https://github.com/Originate/check-protocols/issues)
-[![open pull requests](https://img.shields.io/github/issues-pr-raw/originate/check-protocols.svg?color=%238888ff)](https://github.com/Originate/check-protocols/pulls)
-[![closed issues](https://img.shields.io/github/issues-closed/originate/check-protocols.svg?color=green&label=closed%20issues)](https://github.com/Originate/check-protocols/issues?q=is%3Aissue+is%3Aclosed)
-[![issue board](https://img.shields.io/badge/check--protocols-issue%20board-important.svg)](https://github.com/orgs/Originate/projects/1)
+# `scriptkeeper`
+[![CircleCI](https://circleci.com/gh/Originate/scriptkeeper.svg?style=svg)](https://circleci.com/gh/Originate/scriptkeeper)
+[![open issues](https://img.shields.io/github/issues/originate/scriptkeeper.svg?label=open%20issues)](https://github.com/Originate/scriptkeeper/issues)
+[![open pull requests](https://img.shields.io/github/issues-pr-raw/originate/scriptkeeper.svg?color=%238888ff)](https://github.com/Originate/scriptkeeper/pulls)
+[![closed issues](https://img.shields.io/github/issues-closed/originate/scriptkeeper.svg?color=green&label=closed%20issues)](https://github.com/Originate/scriptkeeper/issues?q=is%3Aissue+is%3Aclosed)
+[![issue board](https://img.shields.io/badge/scriptkeeper-issue%20board-important.svg)](https://github.com/orgs/Originate/projects/1)
```

Comment:
I hope all the links work out, and we should test this once the repo is renamed.